core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -114,6 +123,19 @@ class AdminClient(val time: Time,
     listAllGroupsFlattened.filter(_.protocolType == ConsumerProtocol.PROTOCOL_TYPE)
   }
 
+  def listAllBrokerVersionInfo(): Map[Node, Try[NodeApiVersions]] = {
+    findAllBrokers.map {
+      case broker =>
```

Comment:
Nit: should be in the previous line and `case` is not needed here (if you're feeling generous, you could fix `listAllGroups` as well.